include/irr/asset/IDescriptorSetLayout.h

```diff
@@ -25,6 +25,20 @@ enum E_DESCRIPTOR_TYPE : uint32_t
 	EDT_INVALID = ~0u
 };
 
+//! Interface class for Descriptor Set Layouts
+/*
+	The descriptor set layout specifies the bindings (in the shader GLSL
+	interfaces), counts and types of resources like:
+
+	- UBO
+	- SSBO
+	- combined image samplers
+	- storage images
+
```

Comment:
give examples of how to set up the `SBinding` array for a given GLSL `layout(set=X,binding=Y) descriptorType descriptorName[Z]`

Reply:
adjusted in following commit